src/common/tse_internal.h

```diff
@@ -55,7 +55,9 @@ struct tse_task_private {
 					 dtp_completing:1,
 					/* task is in running state */
 					 dtp_running:1,
-					 dtp_dep_cnt:29;
+					/* Don't propagate err-code from dependent tasks */
+					 dtp_no_propagate:1,
```

Comment:
so not propagate the error of condition check to top task?   it is a bit strange to put it to the general function. Could you please explain a bit? thanks.

Reply:
now only one use case is in dc_tx_get_epoch(), it is for the case that other sub-task depend on first sub-task to get back valid epoch from server-side. in that case the only requirement is get epoch from dep task's reply and should not propagate dep task's err code. 
A specific example is in HDF5 vol-daos code, a few fetch tasks fall in that case, and the first task get REC2BIG error (that is expected) but that should not affect other tasks which only want to get a determined epoch from first task's reply. Without this change the HDF5 vol-daos test cannot pass.